src/js/lib/columnOrder.ts

```diff
@@ -1,13 +1,13 @@
 import {$Column} from "../state/Columns/models/column"
 
-const EXCLUDED = ["ts", "_td"]
+const EXCLUDED = ["ts", "_td", "event_type"]
 
 export default (cols: $Column[]) => {
-  const orderedCols = cols.filter(({name}) => !EXCLUDED.includes(name))
+  let orderedCols = cols.filter(({name}) => !EXCLUDED.includes(name))
   const ts = cols.find(({name}) => name === "ts")
-  if (ts) {
-    return [ts, ...orderedCols]
-  } else {
-    return orderedCols
-  }
+  const eventType = cols.find(({name}) => name === "event_type")
+
+  if (eventType) orderedCols = [eventType, ...orderedCols]
```

Comment:
Moved event_type to the front of the columns after ts. Unfortunately, the way we handle "auto" columns is by creating 1 huge set of columns, then asking each record to display their fields in that order.

It seems like what we should do is, when in "auto" mode, simply render all the fields of each log in the order they were given from the server. This is a larger task than what was feasible here.

This is a little hack to move the event_path to the front just like we do with "_path" and "ts".